src/functionalTest/resources/scenarios/RIA-7582-internal-end-appeal-automatically-notice-document.json

```diff
@@ -0,0 +1,50 @@
+{
+  "description": "RIA-7582 internal end appeal automatically notice document PDF",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "System",
+    "input": {
+      "eventId": "endAppealAutomatically",
+      "state": "*",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "appealSubmissionDate": "{$TODAY-2}",
+          "isAcceleratedDetainedAppeal": "No",
+          "appellantInDetention": "Yes",
+          "tribunalDocuments": [],
+          "isAdmin": "Yes",
+          "remissionType": "noRemission"
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "minimal-appeal-submitted.json",
+      "replacements": {
+        "isAdmin": "Yes",
+        "appellantInDetention": "Yes",
+        "isAcceleratedDetainedAppeal": "No",
+        "notificationAttachmentDocuments": [
+          {
+            "id": "1",
+            "value": {
+              "document": {
+                "document_url": "$/http.+\/documents/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/",
+                "document_binary_url": "$/http.+\/documents/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}\/binary/",
+                "document_filename": "PA 11111 2019-Gonzlez-detained-appellant-request-respondent-evidence-notice.PDF"
```

Comment:
Will the file name be same as request Respondent Evidence notice when the appeal ends automatically?

Reply:
Fixed that